modules/platform/nuxeo-retention-core/src/main/java/org/nuxeo/retention/service/RetentionManagerImpl.java

```diff
@@ -255,7 +255,7 @@ protected ELActionContext initActionContext(DocumentModel doc, CoreSession sessi
     protected boolean evaluateConditionExpression(ELActionContext ctx, String expression) {
         Calendar now = Calendar.getInstance();
         if (StringUtils.isEmpty(expression)) {
-            return true;
+            return false;
```

Comment:
Isn't this dangerous ? The name didn't change and this is not called in this pr and I have no clue to explain why the change. This put asside I'm okay with an "evaluate" method to return false on empty strings

Reply:
no, if we return the `true` as it was before the modification. The expression will be evaluated as true even the expression is empty see [here](https://github.com/nuxeo/nuxeo/blob/3b9e0d5ae01cb24c26738b650a8f2225685fa6a9/modules/platform/nuxeo-retention-core/src/main/java/org/nuxeo/retention/service/RetentionManagerImpl.java#L294)

Reply:
Yes I get that and agree. 
I'm just surprised it was wrong and it's just switched without comment.